Minesweeper.py

```diff
@@ -44,132 +35,133 @@ def __init__(self):
             "The numbers say how many mines are adjacent to the tile cleared",
             "",
             "Press R to restart",
+            "Press ESCAPE to quit"
             "",
             "Press SPACE to start the game.",
         ]
 
-    def draw(self): # draws the tutorial screen
-        # DHB: using win as a global variable is a bad practice. It is better to send
-        # DHB: things like this into constructor args and store in member variables.
-        # DHB: When you start unit testing, global variables are a class of hell
-        # DHB: also, pretty sure you should only fill the window with the background color in the top-level draw function
+    def draw(self):
+        """ draws the tutorial screen using the text information from the init function
+        """
         win.fill(Color.BACKGROUND)
-        for i, instruction in enumerate(self.instructions):
+        for i, instruction in enumerate(self.message):
             text = self.font.render(instruction, True, self.text_color)
             text_rect = text.get_rect(center=(win.get_width() // 2, 100 + i * 30))
             win.blit(text, text_rect)
-        # DHB: I'm pretty certain only the top-level draw function should call display.update 
-        pygame.display.update() # Update the display
 
     def handle_events(self, events):
+        """ causes the transition from the tutorial state to the game state
+        """
```

Comment:
This function 's purpose is to handle input events. That should also be part of its function comment.